practice-one/src/index.html

```diff
@@ -11,6 +11,18 @@
 <body>
     <!--================================= Page Header =================================-->
     <header class="header">
+      <div class="header-container">
+        <div class="logo-section">
+          <a href="#" class="logo-img"><img src="assets/images/header/logo.webp" alt="logo"></a>
+        </div>
+
+        <div class="nav-menu">
```

Comment:
Try using ul, li. Because the navigation menu is like a list. Also you should use `nav` tag for a navigation ui